tests/test_misc.py

```diff
@@ -2101,3 +2103,64 @@ def test_retrieve_vault_status(revault_network, bitcoind):
     mans[0].wait_for_secured_vaults([deposit])
 
     # TODO: same dance with all emergency statuses
+
+@pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
+def test_sigfetcher(revault_network, bitcoind, executor):
+    (stks, mans) = revault_network.deploy(2, 1)
```

Comment:
Can we do a larger setup, like `7, 3` to make it closer to a realistic one which we want to sanity check?

*(yeah, i know, it does not really make sense but y'know, edgecases, esoteric coordinator, all that)*